README.md

```diff
@@ -150,15 +150,15 @@ This looks for the default `.conf` file, to check any file, use:
 		checkip-ssl    = false
         checkip-server = api.ipify.org
     }
-    
+
     # With multiple usernames at the same provider, index with :#
     provider no-ip.com:3 {
         username        = spaceman
         password        = bowie
         hostname        = spaceman.no-ip.com
         checkip-command = "/sbin/ifconfig eth0 | grep 'inet6 addr'"
     }
-    
+
     # Note: hostname == update-key from Advanced tab in the Web UI
```

Comment:
Not 100% sure about these, but if I remember correctly some Markdown implementations require base-level whitespace to be added to empty lines in a verbatim paragraph.  Otherwise it might break formatting.